aepsych/config.py

```diff
@@ -103,7 +104,27 @@ def _get(
                 ),
                 **kwargs,
             )
+    # Convert config into a dictionary (but as it is, excludes "common")
```

Comment:
Is that right? The way ConfigParser works, things that are in `common` are mirrored to all other sections. What this method is doing (I think) is omitting these mirrored copies (basically if something is in `common`, and also shows up in another section, just ignore the one in the other section since it's a mirror).

Reply:
Oh I absolutely did not know that. I knew default sections in ConfigParsers were special but now it makes sense why this line is here!

Reply:
Ok -- can you update the comment to be correct or remove?

Reply:
Yeah I'll update it, haven't pushed any commits yet